packages/svelte-docgen/src/doc/type.ts

```diff
@@ -190,6 +190,8 @@ export interface Union extends WithAlias {
 	nonNullable?: TypeOrRef;
 }
 
+export type AdvancedType = Array | Constructible | Fn | Interface | Intersection | Literal | Tuple | Union;
```

Comment:
Is AdvancedType a term used in TypeScript, or is it something we introduced ourselves? I think we should try to avoid introducing custom terminology whenever possible.

Reply:
Good point.
I'm on mobile (OTW) to airport. Thanks for merging!

I don't yet have idea for a better name.
Detailed? Complex?

Reply:
Aye, that's perfect. Let's name it Structured then.